adapters/sovrn/sovrn.go

```diff
@@ -61,6 +39,54 @@ func (s *SovrnAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapt
 		}
 	}
 
+	errs := make([]error, 0, 1)
```

Comment:
So this will now fail and return on the first error encountered, thus not allowing any valid imps through if any imp is invalid? Is that desired?

Reply:
Do we have any tests around requests with multiple `Imp`s for our adapter?

Reply:
This is something I discussed with John R. Right now we only accept a single `imp` object. This will be opened up when we do the multibid work, which I'm pretty sure is next.

Reply:
We accept multiple IMPs but only one format per imp

Reply:
@jrosendahl Should multiple imps be split into different requests or not?

Reply:
there is no need to do so

Reply:
OK, on it.

Reply:
@sriggin There are now tests for multiple imps. Please re-review.